.github/workflows/ci-macos.yml

```diff
@@ -38,15 +38,15 @@ jobs:
           eval $(opam env)
           ./configure -prefix "$(pwd)/_install_ci" -native-compiler no
           make dunestrap
-          dune build -p coq-core,coq-stdlib,coqide-server,coqide,coq
+          dune build -p coq-core,rocq-init,coqide-server,coqide,coq
```

Comment:
I am surprised that it works. Where does Dune find the `coq-stdlib` dependency of `coq`? Does it use the 8.20.0 one from the main Opam repository? I don't see anything relevant in the logs.

Reply:
If I understand the log, it seems dune just doesn't know about the dependencies added in coq.opam.templates so it just doesn't install coq-stdlib. We should probably remove the coq here for better clarity.